packages/web-components/fast-components/src/listbox/listbox.styles.ts

```diff
@@ -52,21 +51,27 @@ export const listboxStyles: FoundationElementTemplate<ElementStyles> = (
     `.withBehaviors(
         forcedColorsStylesheetBehavior(
             css`
-            :host(:${focusVisible}) ::slotted([aria-selected="true"][role="option"]) {
-                background: ${SystemColors.Highlight};
-                border-color: ${SystemColors.ButtonText};
-                box-shadow: 0 0 0 calc(${focusStrokeWidth} * 1px) inset ${SystemColors.HighlightText};
-                color: ${SystemColors.HighlightText};
-                fill: currentcolor;
-            }
-
-            :host(:${focusVisible}) ::slotted([aria-selected="true"][role="option"]) {
-                background: ${SystemColors.Highlight};
-                border-color: ${SystemColors.ButtonText};
-                box-shadow: 0 0 0 calc(${focusStrokeWidth} * 1px) inset ${SystemColors.HighlightText};
-                color: ${SystemColors.HighlightText};
-                fill: currentcolor;
-            }
-        `
+                :host(:${focusVisible}) ::slotted([aria-selected="true"][role="option"]) {
```

Comment:
massive nit - should these attributes be switched? I think specificity is the same, but ideally this conveys an options which is selected...